playbooks/provision-rh-sso/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- import_playbook: ../prep.yml
+  when:
+  - rhsm_register|default(False)
+  tags:
+  - 'never'
+  - 'install'
+
+- import_playbook: ../osp/manage-user-network.yml
+  when:
+  - hosting_infrastructure == 'openstack'
+  tags:
+  - 'never'
+  - 'install'
+
+- import_playbook: ../osp/provision-osp-instance.yml
+  when:
+  - hosting_infrastructure == 'openstack'
+  tags:
+  - 'never'
+  - 'install'
+
+- import_playbook: ../rhsm.yml
+  tags:
+  - 'never'
+  - 'install'
+
+- hosts: rh-sso-hosts
+  roles:
+  - role: update-host
+  tags:
+  - 'never'
+  - 'install'
+
+- hosts: rh-sso-hosts
+  roles:
+  - role: config-rh-sso
+  tags:
+  - 'never'
+  - 'install'
```

Comment:
Should allow for configuration post-install, so please make sure to not keep these tags here when finalizing this content

Reply:
Ok, noted!